fiftyone/core/threed/splat.py

```
@@ -0,0 +1,115 @@
+
+    Raises:
+        ValueError: if the path or format is not supported
+    """
+
+    _asset_path_fields = ["splat_path"]
+
+    def __init__(
+        self,
+        name: str,
+        splat_path: str,
+        format: Optional[str] = None,
```

Ruff (0.15.21)

[error] Function argument `format` is shadowing a Python builtin

(A002)